content/applications/websites/ecommerce/managing_products/catalog.rst

```diff
@@ -0,0 +1,184 @@
+=======
+Catalog
+=======
+
+The eCommerce catalog is the equivalent of your physical store shelves: it allows customers to see
+what you have to offer. Clear categories, available options, sorting, and navigation threads help
+you structure it efficiently.
+
+Categorize the product catalog
+==============================
+
+In Odoo, there is a **specific category model** for your eCommerce. Using eCommerce categories for
+your products allows you to add a navigation menu on your eCommerce page. Visitors can then use it
+to view all products under the category they select. To do so, go to :menuselection:`Website app
+--> eCommerce --> Products`, select the product you wish to modify, click on the :guilabel:`Sales`
+tab. and select the :guilabel:`Categories` you want under :guilabel:`eCommerce Shop`.
+
+.. note::
+   A single product can appear under multiple eCommerce categories.
+
+When your product's categories are configured, go to :menuselection:`Website dashboard --> Shop` and
+click on :menuselection:`Edit --> Customize tab`. In the :guilabel:`Categories` option, you can
```

Comment:
There is no website dashboard from Odoo 16. Also, "shop" could be named differently, or placed differently in the top menu than how it is by default with the demo data.
I suggest
"Go to your shop page" or something like that

Reply:
same thing below

Reply:
When I read the other iterations in this doc, I think the easiest is to write
"Start editing your [main?] shop page and open the xxx tab"

Reply:
Changed it all